cibuildwheel/docker_container.py

```diff
@@ -88,12 +201,24 @@ def __exit__(
     ) -> None:
 
         self.bash_stdin.close()
+
+        if self._sleep_time:
+            time.sleep(self._sleep_time)
+
         self.process.terminate()
```

Comment:
```suggestion
```

I believe the error is this call itself, since we exhausted the last handle(stdin) and we did not specify detach? Hopefully it is reproducible with your conf.

Reply:
That does seem to work on my local tests. I'm going to remove the sleep and give it a try on the gitlab instance.